dijet/scripts/convert_to_root.py

```diff
@@ -42,17 +42,18 @@
     print(f"{arg}: {getattr(args, arg)}")
 
 # Construct the file path
-base_path = "/nfs/dust/cms/user/paaschal/WorkingArea/Analysis/JERC/DiJet/data/dijet_store/analysis_dijet/"
+base_path = os.getenv("CF_STORE_LOCAL")
 full_path = os.path.join(
     base_path,
-    "dijet.JERtoRoot/",
-    f"{args.config}/",
-    f"{args.shift}/",
-    f"calib__{args.uncertainty}/",
+    "analysis_dijet",
+    "dijet.JERtoRoot",
+    f"{args.config}",
+    f"{args.shift}",
+    f"calib__{args.uncertainty}",
```

Comment:
Only just noticed this, but why is the argument for the calibrator part called `uncertainty`? I would suggest to call it `--calibrator` or similar.